bc_obps/compliance/models/elicensing_payment.py

```diff
@@ -31,6 +31,12 @@ class ElicensingPayment(TimeStampedModel):
         blank=True,
     )
 
+    method = models.CharField(
+        db_comment="The method used to make the payment (e.g. wire transfer) from elicensing.", blank=True, null=True
+    )
+
+    receipt_number = models.CharField(db_comment="Receipt number of the payment in elicensing.", blank=True, null=True)
+
```

Comment:
Can a payment record exist before a payment is actually made? I assumed yes because received_date is optional.